mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -490,6 +490,12 @@ void CMomentumPlayer::Spawn()
             ClearStartMark(i, false);
         }
 
+        // Load startmarks after player spawn
+        if(g_pSavelocSystem->LoadStartMarks())
```

Comment:
Spacing between the if and the parenthesis.

Reply:
This is now fixed.